src/utils/SdkCoin.sol

```diff
@@ -90,13 +105,13 @@ library SdkCoin {
      * @param amount The amount to be converted
      * @return convertedAmount The amount converted to uint256 supported by ERC20 tokens
      */
-    function _convertSdkCoinAmountToERC20(address tokenAddress, uint64 amount) internal view returns (uint256) {
+    function _SdkCoinToERC20_ConvertAmount(address tokenAddress, uint64 amount) internal view returns (uint256) {
         // Get the token decimals
         // address input validation perfomed in the _getERC20TokenDecimals
         uint8 tokenDecimals = _getERC20TokenDecimals(tokenAddress);
 
-        // Ensure the token has at least 6 decimals
-        if (tokenDecimals < 6) {
+        // Ensure the token has at least 6 decimals and max 77
+        if (tokenDecimals < 6 || tokenDecimals > 77) {
```

Comment:
Let's not use hard coded constants. Instead, define them at the top of the file (like `DEFAULT_COSMOS_DECIMALS `), and then use these as named constants.

Reply:
This is not yet addressed